lib/ASTGen/Sources/ASTGen/ASTGen.swift

```diff
@@ -150,106 +133,255 @@ extension ASTGenVisitor {
+  func generate(_ node: CodeBlockItemListSyntax) -> BridgedArrayRef {
+    node.lazy.map { self.generate($0).bridged }.bridgedArray(in: self)
+  }
+    
+  public func generate(_ node: Syntax) -> ASTNode {
+    switch node.as(SyntaxEnum.self) {
+    case .actorDecl(let node):
+      return generate(node)
+    case .arrayElement(let node):
+      return generate(node)
+    case .arrayExpr(let node):
+      return generate(node)
+    case .arrayType(let node):
+      return generate(node)
+    case .associatedTypeDecl(let node):
+      return generate(node)
+    case .attributedType(let node):
+      return generate(node)
+    case .booleanLiteralExpr(let node):
+      return generate(node)
+    case .classDecl(let node):
+      return generate(node)
+    case .closureExpr(let node):
+      return generate(node)
+    case .codeBlock(let node):
+      return generate(node)
+    case .codeBlockItem(let node):
+      return generate(node)
+    case .compositionType(let node):
+      return generate(node)
+    case .conditionElement(let node):
+      return generate(node)
+    case .declReferenceExpr(let node):
+      return generate(node)
+    case .deinitializerDecl(let node):
+        return generate(node)
+    case .dictionaryType(let node):
+      return generate(node)
+    case .enumCaseDecl(let node):
+      return generate(node)
+    case .enumCaseElement(let node):
+      return generate(node)
+    case .enumCaseParameter(let node):
+      return generate(node)
+    case .enumCaseParameterClause(let node):
+      return generate(node)
+    case .enumDecl(let node):
+      return generate(node)
+    case .expressionStmt(let node):
+      return generate(node)
+    case .extensionDecl(let node):
+      return generate(node)
+    case .functionCallExpr(let node):
+      return generate(node)
+    case .functionDecl(let node):
+      return generate(node)
+    case .functionParameter(let node):
+      return generate(node)
+    case .functionParameterClause(let node):
+      return generate(node)
+    case .functionType(let node):
+      return generate(node)
+    case .genericParameter(let node):
+      return generate(node)
+    case .genericParameterClause(let node):
+      return generate(node)
+    case .genericWhereClause(let node):
+      return generate(node)
+    case .identifierPattern(let node):
+      return generate(node)
+    case .identifierType(let node):
+      return generate(node)
+    case .ifExpr(let node):
+      return generate(node)
+    case .implicitlyUnwrappedOptionalType(let node):
+      return generate(node)
+    case .importDecl(let node):
+      return generate(node)
+    case .initializerClause(let node):
+      return generate(node)
+    case .initializerDecl(let node):
+      return generate(node)
+    case .integerLiteralExpr(let node):
+      return generate(node)
+    case .labeledExprList(let node):
+      return generate(node, leftParen: nil, rightParen: nil)
```

Comment:
We should not ever reach this case because this is not a standalone `generate` method, it’s always called directly inside other `generate` methods.